src/components/pages/SystemPage.jsx

```diff
@@ -79,7 +79,7 @@ const SystemPage = (props) => {
         <Nav>
           {menu}
         </Nav>
-        <div className="content overflow-x-auto pb-0">
+        <div className="content pb-0">
```

Comment:
this change did not break any page on the platform?

Reply:
No, I checked